src/components/BookList.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="book-grid">
+    <div class="book-container" v-for="book in books">
```

Comment:
Add a `key` binding every time you make a list render with `v-for` https://vuejs.org/v2/guide/list.html#key